widget/widget.go

```diff
@@ -112,16 +111,14 @@ func (w *BaseWidget) Hide() {
 
 // Refresh causes this widget to be redrawn in its current state
 func (w *BaseWidget) Refresh() {
-	async.EnsureMain(func() {
-		impl := w.super()
-		if impl == nil {
-			return
-		}
+	impl := w.super()
+	if impl == nil {
+		return
+	}
 
-		w.themeCache = nil
```

Comment:
This change does not look like it belongs here?

Reply:
The commit comment "Widget.Refresh running thread handling code was not great for potential loop scenarios" explains it I think...

Reply:
The added bonus being that you the long log of thread warnings from the test driver that were being complained about is gone.

Reply:
Ah, sorry